Marlin/Configuration.h

```diff
@@ -1485,6 +1494,11 @@
   #define U8GLIB_SH1106
 #endif
 
+//#define CR10_stockdisplay
```

Comment:
Sorry if this isn't obvious, but configuration defines must be `ALL_CAPS`.

Reply:
That said… This is just a standard reprap display. We don't want to have a separate entry for every machine that happens to have slightly different timing for its display. If you think that special `ST7920_DELAY_#` values are needed, they should be placed in the CR-10 example configurations, and this should not define a "faux" display type.